stdlib/pwd.pyi

```diff
@@ -1,17 +1,31 @@
-from typing import ClassVar, Tuple
+from _typeshed import structseq
+from typing import Any, Sequence, overload
+from typing_extensions import Literal, SupportsIndex, final
 
-class struct_passwd(Tuple[str, str, int, int, str, str, str]):
-    pw_name: str
-    pw_passwd: str
-    pw_uid: int
-    pw_gid: int
-    pw_gecos: str
-    pw_dir: str
-    pw_shell: str
-
-    n_fields: ClassVar[int]
-    n_sequence_fields: ClassVar[int]
-    n_unnamed_fields: ClassVar[int]
+@final
+class struct_passwd(structseq[Any]):  # Constructor must be passed an iterable of length 7
```

Comment:
I'm arfaid these comments might get outdated easily.

Reply:
But it doesn't error if the comment says 11, and somewhere 30 lines below there's a 12th member.

Reply:
Reckon I should delete them all, then, and just have one comment in the `structseq` definition? I worry people won't see it there if they're browsing the stubs, and think it's important to understand

Reply:
To me it seems like something that I would care about only if I wonder what a `structseq` is, so it belongs near `class structseq`. Most struct sequences don't have to override the constructor.

Reply:
Alright, I'll take them out, then.